src/test/scala/org/apache/spark/sql/delta/LogStoreSuite.scala

```diff
@@ -271,6 +271,18 @@ class HDFSLogStoreSuite extends LogStoreSuiteBase {
   protected def shouldUseRenameToWriteCheckpoint: Boolean = true
 }
 
+class GcsLogStoreSuite extends LogStoreSuiteBase {
```

Comment:
How this test works if we didn't add GCS connector as a Delta dependency? Does Delta has any integration tests that can verify that this implementation is correct?

Reply:
These are unit tests. Integration tests are triggered by running run-integration-tests.py which simply run scripts from examples/ (https://github.com/delta-io/delta/tree/master/examples)
I've run python scripts from examples/ on a dataproc cluster through pyspark after some minor modifications. So didn't need to provide dependency. Also dependencies for aws or azure isn't provided in sbt, so i'm guessing this is standard.